knowledge/astronomy/constellations/Phoenix/qna.yaml

```diff
@@ -0,0 +1,19 @@
+created_by: flyingbarron
+domain: constellation
+seed_examples:
+- question: When was the constellation Phoenix defined?
+  answer: | It first appeared on a 35-cm diameter celestial globe published in 1597 (or 1598) in Amsterdam by Petrus Plancius with Jodocus Hondius. It was first depicted on a celestial atlas by Johann Bayer in his 1603 Uranometria. The French explorer and astronomer Nicolas Louis de Lacaille charted the brighter stars and gave their Bayer designations in 1756.
```

Comment:
Thank you for your submission; taking a look at your `qna.yaml`, we noticed that you have some issues with the multiline `yaml` syntax.

Can you take a look at: https://yaml-multiline.info/ and fix up the file? Thanks!

Reply:
Done, I think.